lib/motion-spec/matcher/satisfy.rb

```diff
@@ -0,0 +1,20 @@
+# -*- encoding : utf-8 -*-
+module MotionSpec
+  module Matcher
+    class Satisfy
+      def initialize(&condition_block)
+        @condition_block = condition_block
+      end
+
+      def matches?(*values)
+        @condition_block.call(*values)
+      end
+
+      def fail!(subject, negated)
```

Comment:
Unused method argument - `subject`. If it's necessary, use `_` or `_subject` as an argument name to indicate that it won't be used.